pkg/cosign/verify.go

```diff
@@ -602,27 +607,31 @@ func verifySignatures(ctx context.Context, sigs oci.Signatures, h v1.Hash, co *C
 	signatures := make([]oci.Signature, len(sl))
 	bundlesVerified := make([]bool, len(sl))
 
-	var wg sync.WaitGroup
-
+	t := throttler.New(co.MaxWorkers, len(sl))
```

Comment:
We'll need to add this to [verifyImageAttestations](https://github.com/sigstore/cosign/blob/main/pkg/cosign/verify.go#L950) as well!